### PR TITLE
New editor

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: src/components/Routing/constants.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Login from 'screens/Auth/screens/Login';
 import Products from 'screens/Products';
 import Logout from 'screens/Auth/screens/Logout';
 import buyManagement from 'screens/BuyManagement';
+import Playground from 'screens/Playground';
```
Comment (nathan-hyan) [inline src/components/Routing/constants.ts:6]: remove playground

File: src/components/Routing/constants.ts [hunk 2]
```diff
@@ -32,5 +33,10 @@ export const ROUTES = [
     route: '/logout',
     component: Logout,
   },
+  {
+    id: 5,
+    route: '/playground',
+    component: Playground,
+  },
```
Comment (nathan-hyan) [inline src/components/Routing/constants.ts:40]: same here

File: src/screens/Playground/index.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const parsedHTML = parse(value); // Esto es lo que toma el string del html y lo hace en DOM
+
+  return (
+    <div>
+      <ReactQuill theme="snow" value={value} onChange={setValue} />
+      {parsedHTML}
+      {/* Acá ya es el html que se convirtió desde string */}
+    </div>
+  );
+}
```
Comment (nathan-hyan) [inline src/screens/Playground/index.tsx:18]: delete

**Review by nathan-hyan — APPROVED** [review]

File: src/components/Routing/constants.ts (hunk 1 above)
Comment (JaqKent) [reply src/components/Routing/constants.ts:6]: done

File: src/components/Routing/constants.ts (hunk 2 above)
Comment (JaqKent) [reply src/components/Routing/constants.ts:40]: done

File: src/screens/Playground/index.tsx (hunk 3 above)
Comment (JaqKent) [reply src/screens/Playground/index.tsx]: done

**Pull request merged**